man/render.Rd

```diff
@@ -107,8 +114,8 @@ Additional format options can also be specified in metadata. For example:
 \preformatted{
 output:
   pdf_document:
-    toc: true
-    highlight: zenburn
+  toc: true
+highlight: zenburn
```

Comment:
It is great to move Rd to roxygen, but you need to be a little cautious about the `\preformatted` tags, in which white spaces are actually meaningful. For example, here the indentation is wrong.